libs/icu_headroom_metric.py

```diff
@@ -17,6 +17,10 @@
 
 DEFAULT_ICU_DECOMP = 0.21
 
+NJ_CORRECTION = (
+    0.176 - 0.57 + DEFAULT_ICU_DECOMP
```

Comment:
I'm a bit confused by the subtraction here.  Could you update the comment to mention the mechanics of the subtraction? maybe something that explains the effect of this being negative

Reply:
Added a comment.